src/schema/schema.graphql

```diff
@@ -1734,6 +1734,16 @@ input AttachmentInput {
 	existingAttachments: [ExistingAttachmentInput]
 }
 
+input DocumentIdInput {
```

Comment:
we should call it DocumentInput
you never know in future you will be passing more data then just id

Reply:
done